content/arduino-cloud/02.hardware/06.device-provisioning/content.md

```diff
@@ -125,11 +129,65 @@ Now your board will be updated to version 2.0, wait for the process to finish.
 
 When this is done you can continue setting up the network connection for your board and it will now be using the latest provisioning method with the Arduino Cloud!
 
-### Removing Saved Network Configuration
+## How to update the Stored Network Configuration
+
+To proceed with the next steps, the cloud-generated sketch must be uploaded to the board. If you want to update the stored network configuration, there are two ways:
+
+### Update Using the BLE Interface
+
+Ensure that the cloud sketch has enabled the BLEAgent, via `thingProperties.h`. This is enabled by default if the board has been registered with Provisioning 2.0. If you need to enable it, please have a look at [this documentation](https://docs.arduino.cc/arduino-cloud/cloud-interface/sketches).
+
+You can update the network settings from the detail page of the device on the Arduino Cloud webpage or the mobile app.
+
+During this process you will be asked to wipe out the current network configuration to restart the board's BLE interface. 
+
+1. Power on the board.
+2. Open the devices page of the Mobile App or the Arduino Cloud webpage.
+3. Click on the device you want to update the network settings for.
+4. Click on the "change" button by the network section.
+5. If you are using the Arduino Cloud webpage, select the BLE method.
+6. Reset the board using the method of your board. Have a look at the [How to set up the Reconfiguration Procedure](#how-to-set-up-the-reconfiguration-procedure) section of this article to find the default board reset pin. If you have changed the reset pin from the default one, please use that.
+7. The board will reboot, and you will see the LED pulsing.
+8. Connect to the board.
```

Comment:
What does this mean? Connect how?

Reply:
There is a "modal" that asks you to choose the device to connect with. It's similar to the provisioning phase